src/keyframe-interpolations.js

```diff
@@ -45,7 +45,7 @@
         if (member != 'offset' && member != 'easing' && member != 'composite') {
           var propertySpecificKeyframe = {
             offset: keyframes[i].offset,
-            easing: keyframes[i].easing,
+            easing: keyframes[i][shared.easingFunctionSymbol],
```

Comment:
I am uneasy that this file is using "easing" for the name of the keyframe attribute, while it contains a function. I know this is what it was doing before, but I think we should be consistent about "easing" containing the string value and "_easingFunction" being the function equivalent.